org.eyecreate.vehiclevoyage.json

```diff
@@ -0,0 +1,38 @@
+{
+  "id": "org.eyecreate.vehiclevoyage",
+  "runtime": "org.kde.Platform",
+  "command": "vehiclevoyage",
+  "runtime-version": "5.14",
+  "sdk": "org.kde.Sdk",
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=x11",
+    "--socket=wayland",
+    "--device=dri",
+    "--filesystem=home",
```

Comment:
is there a reason you need access to the whole home directory rather than using the portal or scoping to a specific path or directory?

Reply:
I don't need it at the moment(I only write/read files to Qt AppDataLocation), but thought I might need to have user saved files later. A quick look at portals shows I'm unfamiliar with everything flatpak offers and shouldn't need whole home permission. I'll update the manifest.

Reply:
If you need a permission later you can add it to the manifest then, but certainly always try and ask for the minimum!